packages/platform-sdk-ada/src/services/client.ts

```diff
@@ -27,15 +28,24 @@ export class ClientService implements Contracts.ClientService {
 	}
 
 	public async __destruct(): Promise<void> {
-		//
+		throw new Error("Method not implemented.");
```

Comment:
This is always called and is allowed to be empty and do nothing. Some coins need this to terminate their websocket connections.